inst/testcode.R

```diff
@@ -10,12 +10,22 @@ ui <- fluidPage(
 
 server <- function(input, output) {
   output$timeline <- renderDaisyTimeline({
-    daisyTimeline(list(
-      list(date = "2022", content = "Planning phase"),
-      list(date = "2023", content = "Development started"),
-      list(date = "2024", content = "Launch 🚀"),
-      list(date = "2025", content = "IPO 🚀")
-    ))
+    # Using data frame format (recommended)
+    events_df <- data.frame(
+      date = c("2022", "2023", "2024", "2025"),
+      content = c("Planning phase", "Development started", "Launch 🚀", "IPO 🚀"),
+      stringsAsFactors = FALSE
+    )
+    
+    daisyTimeline(events_df)
+    
+    # Alternative: still works with list format for backward compatibility
+    # daisyTimeline(list(
+    #   list(date = "2022", content = "Planning phase"),
+    #   list(date = "2023", content = "Development started"),
+    #   list(date = "2024", content = "Launch 🚀"),
+    #   list(date = "2025", content = "IPO 🚀")
+    # ))
```

Comment:
Let's get rid of any commented out code - we don't need this backwards compatibility anyway, but we should either include it or delete it.